property.go

```diff
@@ -0,0 +1,93 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string `json:"name"`
+	Schema JSON   `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+	Name    string   `graphql:"name"`
+	Schema  JSON     `graphql:"schema" scalar:"true"`
+}
+
+type PropertyDefinitionConnection struct {
+	Nodes      []PropertyDefinition
+	PageInfo   PageInfo
+	TotalCount int `graphql:"-"`
+}
+
+func (client *Client) CreatePropertyDefinition(input PropertyDefinitionInput) (*PropertyDefinition, error) {
+	var m struct {
+		Payload struct {
+			Definition PropertyDefinition `graphql:"definition"`
+			Errors     []OpsLevelErrors   `graphql:"errors"`
+		} `graphql:"propertyDefinitionCreate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v, WithName("PropertyDefinitionCreate"))
+	return &m.Payload.Definition, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) GetPropertyDefinition(input string) (*PropertyDefinition, error) {
+	var q struct {
+		Account struct {
+			Definition PropertyDefinition `graphql:"propertyDefinition(input: $input)"`
+		}
+	}
+	identifier := *NewIdentifier(input)
+	v := PayloadVariables{
+		"input": identifier,
+	}
```

Comment:
```suggestion
        v := PayloadVariables{
            "input": *NewIdentifier(input),
        }
```
nit: This is more in line with how `PayloadVariables` is defined in `infra.go`, `system.go`, and a couple others